app/controllers/users_controller.rb

```diff
@@ -42,11 +42,24 @@ def csv_import
       Rails.logger.info "***"
       Rails.logger.info "Ingesting CSV file"
 
+      if params[:csv].original_filename.split('.').last.to_s.downcase != 'csv'
```

Comment:
Ideally we would use actual MIME type checking here, rather than just filename extension checking, but it's likely not worth the additional dependencies it would require.